### ISSUE TITLE
Workflow overwrites work directory: warn users

### ISSUE THREAD

**Issue by andrewgettelman** [NOAA-GFDL/MDTF-diagnostics#66]
As discussed in the mini-leads meeting on Nov 18th: the 'wkdir' gets overwritten each time the package as run, and it is only for the output of the package. This was not initially clear to me, and I put input data (model_data and obs_data) in the wkdir.  I recognize this is not optimal. 

Given that a developer may run the package repeatedly on many cases, should the output directory be named with the case in some way, rather than wkdir? I think this could be done in the script perhaps with just wkdir = $CASE_output or something like that. 

This issue is mostly for discussion....

**Comment by jkrasting** [comment]
Thanks @andrewgettelman.  We have included a suggested directory structure in the documentation:

```
mdtf
├── MDTF-diagnostics
├── inputdata
    ├── model ( = $MODEL_DATA_ROOT)
    │   ├── GFDL.CM4.c96L32.am4g10r8
    │   │   └── day
    │   │       ├── GFDL.CM4.c96L32.am4g10r8.precip.day.nc
    │   │       └── (... other .nc files )
    │   └── QBOi.EXP1.AMIP.001
    │       ├── 1hr
    │       │   ├── QBOi.EXP1.AMIP.001.PRECT.1hr.nc
    │       │   └── (... other .nc files )
    │       ├── 3hr
    │       │   └── QBOi.EXP1.AMIP.001.PRECT.3hr.nc
    │       ├── day
    │       │   ├── QBOi.EXP1.AMIP.001.FLUT.day.nc
    │       │   └── (... other .nc files )
    │       └── mon
    │           ├── QBOi.EXP1.AMIP.001.PS.mon.nc
    │           └── (... other .nc files )
    └── obs_data ( = $OBS_DATA_ROOT)
        ├── (... supporting data for individual PODs )
```

@andrewgettelman's suggestion is good.  At a minimum, a check to see if `wkdir` contains any contents and some logic to handle user preference would be good.

**Comment by andrewgettelman** [comment]
This is good, but wkdir is not in this structure. That's what tripped me up. I think it's inside of MDTF-diagnostics here? 
And the big problem I had was that our systems have a small directory for backed up files like code, but storing GB of model output doesn't work there, so I have to point 'inputdata' to another partition on the file system. wrkdir also would go there as it is big. 

Does that make sense? So maybe the solution is just noting where wrkdir ends up, and how to save it's contents.

**Comment by jkrasting** [comment]
I agree - that's a great idea.  

Should we sneak a documentation edit into #68?

**Comment by tsjackson-noaa** [comment]
Closing this issue since it has been implemented as of the [v3.0 beta 3](https://github.com/NOAA-GFDL/MDTF-diagnostics/releases/tag/v3.0-beta.3) release. See PR NOAA-GFDL/MDTF-diagnostics#147.
